huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go

```diff
@@ -29,15 +29,25 @@ import (
 
 // ResourceCCEClusterV3 defines the CCE cluster resource schema and functions.
 // Deprecated: It's a deprecated function, please refer to the function 'ResourceCluster'.
-// @API CCE DELETE /api/v3/projects/{project_id}/clusters/{id}
-// @API CCE GET /api/v3/projects/{project_id}/clusters/{id}
-// @API CCE PUT /api/v3/projects/{project_id}/clusters/{id}
+
 // @API CCE POST /api/v3/projects/{project_id}/clusters
```

Comment:
it seems that we should add those API comments before #L30

Reply:
fixed

Reply:
fixed